docs/Gemfile

```diff
@@ -0,0 +1,35 @@
+source "https://rubygems.org"
+# Hello! This is where you manage which Jekyll version is used to run.
```

Comment:
No need to include the help comments in this file I think.

Reply:
The help comments is removed.